From 27b56a3cf9088208a5aa545ba5d2da7648e46994 Mon Sep 17 00:00:00 2001
From: Danylo Storozhev <office@storozhev.net>
Date: Fri, 23 May 2025 16:50:46 +0200
Subject: [PATCH 1/2] remove redundant target

---
 src/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 2f8acdfc9..3e3f8b8ba 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -234,12 +234,6 @@ if(BUILD_SHARED_LIBS)
   install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${LIBVALHALLA_SO_LINK}
     DESTINATION "${CMAKE_INSTALL_LIBDIR}"
     COMPONENT development)
-
-  if(UNIX)
-    add_custom_target(valhalla-symlink ALL
-      COMMAND ${CMAKE_COMMAND} -E create_symlink "${LIBVALHALLA_SO_LINK}.${VALHALLA_VERSION_MAJOR}" ${LIBVALHALLA_SO_LINK})
-    set_target_properties(valhalla-symlink PROPERTIES FOLDER "Library")
-  endif()
 endif()
 
 install(FILES ${VALHALLA_SOURCE_DIR}/COPYING ${VALHALLA_SOURCE_DIR}/CHANGELOG.md ${VALHALLA_SOURCE_DIR}/README.md
-- 
2.53.0

