From ba8855b8d8fcd301c0f80ccf43ed1b5a4569b815 Mon Sep 17 00:00:00 2001
From: Danylo Storozhev <office@storozhev.net>
Date: Fri, 23 May 2025 17:59:31 +0200
Subject: [PATCH 2/2] set protoc executable

Upstream-Status: Pending
---
 proto/CMakeLists.txt | 26 +++++++++++++++++++++++++-
 1 file changed, 25 insertions(+), 1 deletion(-)

diff --git a/proto/CMakeLists.txt b/proto/CMakeLists.txt
index 6b6792cf5..d31a53666 100644
--- a/proto/CMakeLists.txt
+++ b/proto/CMakeLists.txt
@@ -15,7 +15,31 @@ set(protobuf_descriptors
   isochrone.proto
   expansion.proto)
 
-protobuf_generate_cpp(protobuf_srcs protobuf_hdrs ${protobuf_descriptors})
+set(protobuf_srcs "")
+set(protobuf_hdrs "")
+set(protobuf_descriptors_absolute "")
+
+foreach(proto_file ${protobuf_descriptors})
+  get_filename_component(proto_name ${proto_file} NAME_WE)
+  if(IS_ABSOLUTE ${proto_file})
+    set(proto_abs_path ${proto_file})
+  else()
+    get_filename_component(proto_abs_path "${CMAKE_CURRENT_SOURCE_DIR}/${proto_file}" ABSOLUTE)
+  endif()
+  list(APPEND protobuf_srcs "${CMAKE_CURRENT_BINARY_DIR}/${proto_name}.pb.cc")
+  list(APPEND protobuf_hdrs "${CMAKE_CURRENT_BINARY_DIR}/${proto_name}.pb.h")
+  list(APPEND protobuf_descriptors_absolute ${proto_abs_path})
+endforeach()
+
+add_custom_command(
+  OUTPUT ${protobuf_srcs} ${protobuf_hdrs}
+  COMMAND ${Protobuf_PROTOC_EXECUTABLE}
+  ARGS --cpp_out=${CMAKE_CURRENT_BINARY_DIR}
+       -I${VALHALLA_SOURCE_DIR}/proto
+       ${protobuf_descriptors_absolute}
+  DEPENDS ${protobuf_descriptors_absolute}
+  COMMENT "Generating C++ sources and headers from .proto files using custom command"
+  VERBATIM)
 
 valhalla_module(NAME proto
   SOURCES
-- 
2.53.0

