From d3a02219d96abb85ac7a45134dfae907b9705019 Mon Sep 17 00:00:00 2001
From: Danylo Storozhev <office@storozhev.net>
Date: Mon, 14 Oct 2024 10:23:07 +0000
Subject: [PATCH] mender malloc patch

---
 tools/env/fw_env_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/env/fw_env_main.c b/tools/env/fw_env_main.c
index 443de36e437..aae1cb8be89 100644
--- a/tools/env/fw_env_main.c
+++ b/tools/env/fw_env_main.c
@@ -244,7 +244,7 @@ int main(int argc, char *argv[])
 	argv += optind;
 
 	if (env_opts.lockname) {
-		lockname = malloc(sizeof(env_opts.lockname) +
+		lockname = malloc(strlen(env_opts.lockname) +
 				sizeof(CMD_PRINTENV) + 10);
 		if (!lockname) {
 			fprintf(stderr, "Unable allocate memory");
-- 
2.34.1

