From 6132bb63822fadb1e9fa1f11413a0ea46c737fca Mon Sep 17 00:00:00 2001
From: Danylo Storozhev <office@storozhev.net>
Date: Mon, 14 Oct 2024 10:59:16 +0000
Subject: [PATCH] add kernel_addr_r var

---
 include/configs/mx6ul_14x14_evk.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mx6ul_14x14_evk.h b/include/configs/mx6ul_14x14_evk.h
index bf4653e7b94..fdb5479dec4 100644
--- a/include/configs/mx6ul_14x14_evk.h
+++ b/include/configs/mx6ul_14x14_evk.h
@@ -133,6 +133,7 @@
 	"initrd_high=0xffffffff\0" \
 	"fdt_file=undefined\0" \
 	"fdt_addr=0x83000000\0" \
+	"kernel_addr_r=0x80800000\0" \
 	"tee_addr=0x84000000\0" \
 	"tee_file=undefined\0" \
 	"boot_fdt=try\0" \
-- 
2.34.1

