From 769cf2a8869561505ace6cd14f10b10008e6c7ab Mon Sep 17 00:00:00 2001
From: Danylo Storozhev <daniel@storozhev.net>
Date: Tue, 7 Jan 2025 18:19:47 +0000
Subject: [PATCH] add console boot arg

---
 include/configs/mx6sabre_common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mx6sabre_common.h b/include/configs/mx6sabre_common.h
index fe1fbb142ac..df752bca617 100644
--- a/include/configs/mx6sabre_common.h
+++ b/include/configs/mx6sabre_common.h
@@ -150,6 +150,7 @@
 	"fdt_file=undefined\0" \
 	"fdt_addr=0x18000000\0" \
 	"kernel_addr_r=0x12000000\0" \
+	"bootargs=console=ttymxc0,115200\0" \
 	"tee_addr=0x20000000\0" \
 	"tee_file=undefined\0" \
 	"boot_fdt=try\0" \
-- 
2.34.1

