From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Teal Bauer <teal@librescoot.org>
Date: Mon, 7 Apr 2026 12:00:00 +0200
Subject: [PATCH] center boot logo on display

Upstream-Status: Inappropriate [embedded platform config]
---
 include/configs/mx6sabre_common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mx6sabre_common.h b/include/configs/mx6sabre_common.h
index df752bca617..000000000000 100644
--- a/include/configs/mx6sabre_common.h
+++ b/include/configs/mx6sabre_common.h
@@ -150,6 +150,7 @@
 	"fdt_file=undefined\0" \
 	"fdt_addr=0x18000000\0" \
 	"kernel_addr_r=0x12000000\0" \
+	"splashpos=m,395\0" \
 	"boot_animation=librescoot\0" \
 	"bootargs=console=ttymxc0,115200 loglevel=0 quiet systemd.show_status=false vt.global_cursor_default=0 boot.animation=${boot_animation}\0" \
 	"tee_addr=0x20000000\0" \
